src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -1554,3 +1554,27 @@ function Test-ApplicationGatewayCRUDSubItems2
 		Clean-ResourceGroup $rgname
 	}
 }
+
+function Test-AvailableServerVariables
+{
+	$result = Get-AzApplicationGatewayAvailableServerVariables
```

Comment:
@nirkum18 for the tests you're adding, we need proper test setup so that other users will be able to record this test (including the Network test recording job that we use whenever an SDK is released with a new API version) -- running just these cmdlets will fail if the user does not have the appropriate setup in their subscription, so you should do the necessary resource construction at the beginning of the test and deconstruction at the end of the test. Please see the other tests in this file for an example of what this should look like.

Reply:
@cormacpayne these tests don't need any setup, all they do is query backend for a list of predefined values